package/skelcd-control-MicroOS.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Tue Apr 29 16:01:51 UTC 2025 - Stefan Schubert <schubi@suse.de>
+
+- Disabling kdump
+- Set default bootloader to systemd-boot
```

Comment:
I think it will need some jsc or bsc to get autosubmitted.

Reply:
I have added